diff -uprN debian422/changelog debian423/changelog
--- debian422/changelog	2020-02-03 14:02:10.300723345 +0900
+++ debian423/changelog	2020-02-04 18:47:01.728671320 +0900
@@ -1,3 +1,10 @@
+omniorb (4.2.3-0.1) experimental; urgency=medium
+
+  * Non-maintainer upload.
+  * New upstream version.
+
+ -- Noriaki Ando <n-ando@aist.go.jp>  Mon, 03 Feb 2020 14:21:30 +0100
+
 omniorb-dfsg (4.2.2-0.8) experimental; urgency=medium
 
   * Non-maintainer upload.
diff -uprN debian422/control debian423/control
--- debian422/control	2020-02-03 14:02:10.300723345 +0900
+++ debian423/control	2020-02-04 18:47:13.600671320 +0900
@@ -1,4 +1,4 @@
-Source: omniorb-dfsg
+Source: omniorb
 Section: devel
 Priority: optional
 Maintainer: Debian CORBA Team <pkg-corba-devel@lists.alioth.debian.org>
